src/test/java/SetCollectionTest.java

```diff
@@ -0,0 +1,44 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SetCollectionTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set Size 테스트")
+    public void sizeTest() {
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    @DisplayName("contains 테스트")
+    public void containsTest(int num) {
+        assertThat(numbers.contains(num)).isTrue();
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false", "6:false"}, delimiter = ':')
+    @DisplayName("contains 테스트 V2")
+    public void containsTestV2(int num, boolean result) {
```

Comment:
default delimiter를 사용해도 좋을 것 같습니다 🙂
```suggestion
    @CsvSource(value = {"1,true", "2,true", "3,true", "4,false", "5,false"})
```